NBug/Core/Util/ErrorReportMarkDownBodyBuilder.cs

```diff
@@ -29,6 +29,8 @@ public string Build(Exception exception, string additionalInfo)
     :warning: Review existing issues to see whether someone else has already reported your issue.
 -->
 
+The sections below must be filled in and this text must be removed or the issue will be closed.
```

Comment:
do we want to add :warning: icon?

Reply:
Good idea. 
More important: is this text enough for the bot to close or at least add a label?

Reply:
The dup bot is looking at the title.
We can build another bot that would close issues as incorrectly filled. This text can be used as a marker.